app/views/expenses/_expense.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="container card mb-4">
+	<div class="p-4">
+		<div class="d-flex justify-content-between flex-column">
+			<h5><%= expense.name %></h5>
+			<small>$<%= expense.amount.round(2) %></small>
+			<%= button_to "Delete", expense_path(expense), :method => :delete, :class => "btn btn-danger btn-sm"%>
+		</div>
+		<small class='text-muted'><%= expense.created_at.strftime("%d %b %Y") %></small>
```

Comment:
- [ ] I would recommend keeping the design same as given design for the transactions. It will help in your real work to stick with the design what client or company require.

### Your design
![Screenshot from 2022-04-27 15-30-59](https://user-images.githubusercontent.com/26767383/165499298-653d8b82-cc9a-460a-8e9a-2b4c6f2c92aa.png)



### Given design
![Screenshot from 2022-04-27 15-30-42](https://user-images.githubusercontent.com/26767383/165499257-5b7ed7e5-671a-4713-a708-b59794a16dd4.png)

Reply:
And if i may ask whats so wrong with how i designed it I was only told to pick the colors from that design not the template for it cant be inspected

Reply:
I think look at the project requirements properly

Reply:
@KayLemba, The project requirement includes the layout and design should be the same as original(given) design. Kindly change according to it.
![Screenshot from 2022-04-27 16-51-15](https://user-images.githubusercontent.com/26767383/165508013-ce4d653d-b6f9-4344-aa7e-1b34af0896f6.png)

Reply:
Thank you for the review, but the keyword 'should' in the project doesn't sound like a hard requirement to me, I find that design template (Which is from 2014) very outdated in terms of UI and not worth following, and came up with my own instead, which shouldn't be an issue. The app is behaving as it should, a user has many categories and a category has many transactions, if a category is deleted the transactions are deleted as well.
The only recommended change that makes sense is the failed tests, but that's about it.